glean-core/android/build.gradle

```diff
@@ -65,6 +66,11 @@ android {
     }
 
     sourceSets {
+        main {
```

Comment:
We might not have this documented in a-s, but could you leave a note what this does?